bundles/org.openhab.binding.pushover/README.md

```diff
@@ -73,9 +74,7 @@ Parameter `priority` is the priority to be used (`-2` = lowest priority, `-1` =
 For priority `2` only, the action returns a `String` value (the `receipt`) if the message was sent successfully, otherwise `null`.
 For other priorities, the action always returns an empty `String`.
 
-- `cancelPriorityMessage(String receipt)` - This method is used to cancel an emergency priority message.
-
-The action returns a `Boolean` value to indicate if the message was cancelled successfully or not.
+- `cancelPriorityMessage(String receipt)` - This method is used to cancel an emergency priority message. The action returns a `Boolean` value to indicate if the message was cancelled successfully or not.
```

Comment:
Oh, now I see that the additional line was already there, and this is the reason for the wrong rendering. We can now just remove the excess line, and it should be rendered correctly (like above):
```suggestion
- `cancelPriorityMessage(String receipt)` - This method is used to cancel an emergency priority message.
The action returns a `Boolean` value to indicate if the message was cancelled successfully or not.
```

I don't know why the diff looked different previously.

Reply:
I allready removed the new line couple of minutes ago. This suggestion triues to add it again. Little confusing. But i think we are all good to go as is now.

Reply:
We are. :) And yes, confusing. When I read the diff for the PR earlier today, it seemed like an empty line was added. But actually this additional empty line was already in main, so you didn't add it. And this is the reason for the wrong rendering in the latest docs. We can therefore safely have a line break before the next sentence like for the previous actions.